src/modelmatrix.jl

```diff
@@ -26,7 +26,7 @@ Base.size(mm::ModelMatrix, dim...) = size(mm.m, dim...)
 
 
 ## construct model matrix columns from model frame + name (checks for contrasts)
-function modelmat_cols{T<:AbstractFloatMatrix}(::Type{T}, name::Symbol, mf::ModelFrame; non_redundant::Bool = false)
+function modelmat_cols(T::AbstractFloatMatrix, name::Symbol, mf::ModelFrame; non_redundant::Bool = false)
```

Comment:
Should be `Type{<:AbstractFloatMatrix}`. Anyway `T` is used below so you need to keep it in this case.